src/modules/accolades/AccoladesFormPage.tsx

```diff
@@ -22,68 +21,70 @@ const AccoladesFormPage = () => {
   } = useCreateAccolades();
 
   return (
-    <PagesWrapper>
-      <TitleHeader
-        title={texts.accoladesTitle}
-        description={texts.accoladesDescription}
-      />
+    <div className="flex min-h-screen flex-col">
```

Comment:
same with the height here, getting footer to stick to the bottom
![Screenshot 2024-07-16 at 9 39 36 AM](https://github.com/user-attachments/assets/d8a2a042-0487-48b5-935f-67552b14a065)

Reply:
same ...

Reply:
Updated